### PR TITLE
test

### PR DESCRIPTION
**Pull request by rwike77**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:1]: This will fail validation, as README must follow the required pattern for titles.  _([guideline](https://github.com/Azure/azure-sdk/blob/main/docs/policies/README-TEMPLATE.md))_

- [Example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/README.md)
- [Example](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/README.md)

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:24]: Core 3.1 reaches end of life in December.  _([ref](https://devblogs.microsoft.com/dotnet/net-core-3-1-will-reach-end-of-support-on-december-13-2022/))_ if we're going to recommend a version here, it should be .NET 6+.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:26]: Can you help me understand why this is listed?  Generally, this is not something we include.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:23]: This isn't a prerequisite; other editors such as Rider, Sublime Text, and notepad can be used.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:34]:
```suggestion
* For private preview register a Nuget source either using the NuGet CLI **OR** Visual Studio's NuGet Package Manager.
```

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (jsquire) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:66]:
```suggestion
When running your function in production, it is **highly recommended** to use the [Azure Functions Azure AD authentication integration](https://learn.microsoft.com/azure/app-service/configure-authentication-provider-aad#-option-2-use-an-existing-registration-created-separately) for validating incoming tokens. Set the following function [application settings](https://learn.microsoft.com/azure/azure-functions/functions-how-to-use-azure-function-app-settings?tabs=portal#settings).
```

nit: The rest of the document uses single-spaces after periods.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (ShaneMicro) [inline sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:32]: I don't think we need the private preview instructions anymore as this is in the "released"

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:1]: Fixed.  Thanks, I'll read through the guidelines more closely.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:34]: Fixed, thanks.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:8]: OK, I removed "versioning"

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:32]: OK, I'll remove the Private preview block.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:66]: Ha, sorry.  That's the way I learned to type.  I'll try to be more consistent.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:24]: Removing most of these prereqs, they should be taken care when installing the package.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:23]: Good point, removing.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:32]: Removing most of these prereqs, they should be taken care when installing the package.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:26]: Removing most of these prereqs, they should be taken care when installing the package.

File: sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md
Comment (ShaneMicro) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:16]: Correct, but that is only to list them in VS. you should be able to do a nuget ... -version 1.0.0-beta.1

Comment (rwike77) [reply sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/README.md:16]: Keeping prerelease switch in the readme for the moment, we have plenty of time to change it if need be.

**Comment by rwike77** [comment]
@jsquire  I think we're all set, here.  Anything else I need to do before merging?  Thanks.

**Comment by jsquire** [comment]
> I think we're all set, here. Anything else I need to do before merging? Thanks.

Since @ShaneMicro is the engineering owner, I'd like to have his formal sign-off.   Other than that, I think we're good.

**Comment by ShaneMicro** [comment]
Yup, I am good with the changes ... (Big thumbs up)

**Review by ShaneMicro — APPROVED** [review]

**Review by jsquire — APPROVED** [review]

**Comment by jsquire** [comment]
Alrighty.   We're all merged in.

**Pull request merged**
